Scripts/OrbitCamera.cs

```diff
@@ -78,7 +78,7 @@ private void Update() {
             float deadzone = 0.15f;
             gamepadLook = new Vector2(Mathf.MoveTowards(gamepadLook.x, 0f, 0.15f)/(1f-deadzone),
                                       Mathf.MoveTowards(gamepadLook.y, 0f, 0.15f)/(1f-deadzone));
-            mouseDelta += gamepadLook * 40f;
+            mouseDelta += gamepadLook * 40f * sensitivity;
         }
 
         if (controls != null) {
```

Comment:
Could you remove this section of code if you can confirm it's not used? This is where sensitivity would be involved in the controller input.